language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -121,7 +121,7 @@ describe('decoders for recursive types do not loop', () => {
   // });
```

Comment:
Shouldn’t this pass now?

Reply:
It should indeed. Good catch. I'll make another PR for that.

Reply:
Now you’re just gaming PR stats :trollface:

Reply:
I'm gaming your contribution stats too since code reviews count as well. 🤫